### ISSUE TITLE
How to auto accept new device without press accept request dialog

### ISSUE THREAD

**Issue by Lolmc0587** [nohajc/termux-adb#18]
can we do that?
its really annoying when replug the device

**Comment by nohajc** [comment]
I don't think you can skip the dialog. It's how Android works for non-root users. Security feature...

**Comment by Lolmc0587** [comment]
why termux not remember the previous usb plugged?

**Comment by Lolmc0587** [comment]
https://github.com/kai-morich/SimpleUsbTerminal
this app just support an usb serial but when replug its still working
this app have an remember button
![Screenshot_20230830_183546_Serial USB Terminal](https://github.com/nohajc/termux-adb/assets/59377673/971099d2-7062-47ed-93be-511931574236)

**Comment by nohajc** [comment]
Well, in that case, this is a potential termux improvement. I can't do anything about it directly though...
